src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/IssueDocumentHandlerTest.java

```diff
@@ -195,4 +199,24 @@ public void givenHearingIsNull_thenReturnDraftDecisionNotice() {
         assertEquals(expectedDefaultDocumentLabel, documentTypeLabel);
     }
 
+    @Test
+    public void givenPostHearingReviewIsCorrection_thenUseCorrectionBody() {
+        SscsCaseData sscsCaseData = buildCaseData();
+        sscsCaseData.getPostHearing().setReviewType(PostHearingReviewType.CORRECTION);
+        String bodyContent = "correction body content";
+        sscsCaseData.getDocumentGeneration().setCorrectionBodyContent(bodyContent);
+
+        NoticeIssuedTemplateBody payload = handler.createPayload(null, sscsCaseData, "doctype", LocalDate.now(), LocalDate.now(), false, USER_AUTHORISATION);
+
+        assertEquals(payload.getNoticeBody(), bodyContent);
+    }
+
+    @Test
+    public void givenPostHearingReviewIsLibertyToApply_thenThrowException() {
+        SscsCaseData sscsCaseData = buildCaseData();
+        sscsCaseData.getPostHearing().setReviewType(PostHearingReviewType.LIBERTY_TO_APPLY);
+
+        assertThrows(IllegalArgumentException.class, () ->
+            handler.createPayload(null, sscsCaseData, "doctype", LocalDate.now(), LocalDate.now(), false, USER_AUTHORISATION));
```

Comment:
You can chain an expected message on the end as well, .hasMessage or .hasMessageStartingWith
May need to use a different Assertion method like assertThrownBy.
```suggestion
        assertThrownBy(() ->
            handler.createPayload(null, sscsCaseData, "doctype", LocalDate.now(), LocalDate.now(), false, USER_AUTHORISATION))
            .hasType(IllegalArgumentException.class)
            .hasMessage("...");
```

Reply:
done